### ISSUE TITLE
Respect own accessibility guidelines

### ISSUE THREAD

**Issue by kaelig** [Financial-Times/next-style-guide#43]
For example, `<i>` is used in the markup of the accessibility guidelines page, when it specifically says not to use the tag in core content.

**Comment by keirog** [comment]
Yep we'll be changing that page to include our status in actually adhering to it (you'll probably find that we fail a few others at the moment, at least)
